build/eks/eks_bootstrap_script

```diff
@@ -0,0 +1,165 @@
+#!/bin/bash
+# This script will
+# 1. create an EKS cluster
+# 2. create an external IP
+# 3. add a context to $HOME/.kube/config
+
+set -e
+
+# MacOS does not support readlink, but it does have perl
+KERNEL_NAME=$(uname -s)
+if [ "${KERNEL_NAME}" = "Darwin" ]; then
+  SCRIPT_PATH=$(perl -e 'use Cwd "abs_path";use File::Basename;print dirname(abs_path(shift))' "$0")
+else
+  SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+fi
+
+source "${SCRIPT_PATH}/common.sh"
+
+# Set default values
+# Documentation: doc/installation/cloud/eks.md
+cluster_name=${CLUSTER_NAME-gitlab-cluster}
+nodes=${NUM_NODES-2}
+kubernetes_version=${CLUSTER_VERSION-1.18}
+region="${REGION-us-east-2}"
+node_type="${MACHINE_TYPE-t2.medium}"
+service_account="${SERVICE_ACCOUNT-tiller}"
+kubeconfig="${KUBECONFIG:-$HOME/.kube/config}"
```

Comment:
Are we using the new AWS account assigned for e2e?
If not then lets us that one!

Reply:
Yes, the credentials. i haven't provided in gitlab variables yet. They will be from new account.

Reply:
cool